### ISSUE TITLE
Enhancement idea: Redirect landing page as cmplz_region_redirect auto loads Australian pages

### ISSUE THREAD

**Issue by xhemals** [Really-Simple-Plugins/complianz-gdpr#414]
**Description:**
The company I work for is an international company, so we have the premium version of complainz for all the regions. Sometimes we need to send out links to our privacy policy. When sending users to `website.com/privacy-statement/?cmplz_region_redirect=true` it will automatically load the Australian version of the privacy statement. So the page first loads `website.com/privacy-statement-au/?cmplz_region_redirect=true` before then redirecting the user to their regions privacy statement. This also means any link previews (i.e. when sending a link in MS teams) will have the title `Privacy Statement (AU)` regardless of their region.

This isn't ideal as users think we have sent them the wrong privacy statement. I appreciate that WordPress will (by default) load the AU version as that is the first page in an alphabetical order that is `privacy-statement-*`.  

On our website I have created a workaround for this by created a page with the slug `privacy-statement` so it won't auto load AU. And set up a redirect so every time a user goes to `privacy-statement` it will send them to `privacy-statement/?cmplz_region_redirect=true` with a loading animation to let the user know they are being redirected to their page. This way it doesn't show AU then suddenly switch to their region.

**Steps to Reproduce:**
1. Complianz premium with region redirects turned on
2. Complianz generated privacy statement pages
3. Have no page with the slug `privacy-statement`
4. Go to `website.com/privacy-statement/?cmpls_region_redirect=true`
5. Send link in a messaging application with website previews of `website.com/privacy-statement/?cmpls_region_redirect=true`

**Expected Behavior:**
Loads only privacy statement of users region

**Actual Behavior:**
Loads AU version of privacy statement first

**Additional Context:**
Haven't tested with any of the other complianz generated pages but I assume it will act the same

**Possible Solution:**
Have a landing page that takes the non region slug (i.e. `privacy-statement`) and has content of something like "Loading Privacy Statement" with an animation to show the page is working. Same for other region specific redirected pages.

Let me know your thoughts!

**Comment by rlankhorst** [comment]
@xhemals your solution sounds good. Not sure if this is a good idea to automatically do in the plugin, as it creates an additional page that not everyone wants. 

Maybe a good alternative is an article on the website explaining how to set this up. It has some similarities with the Legal Hub article.

**Comment by xhemals** [comment]
You are right, probably not the best of ideas to do it automatically... Looking through your guys' documentation I think it seems best suited for a guide to go into [How to redirect your policies based on region](https://complianz.io/how-to-redirect-your-policies-based-on-region/). Adding a section like `How to redirect without linking a default region`? and explain how if you go to just /privacy-statement it would normally default to AU. Then take them through the steps to stop this from happening etc.
